src/modules.d.ts

```diff
@@ -33,13 +33,13 @@ declare namespace Electron {
         fromProcessAndFrameIds: (processId: number, frameId: number) => WebContents;
         getOwnerBrowserWindow: () => BrowserWindow | void;
         mainFrameRoutingId: number;
-        session: Session;
+        readonly session: Session;
         registerIframe: (frameName: string, frameRoutingId: number) => void;
         unregisterIframe: (frameName: string, frameRoutingId: number) => void;
     }
 
     export interface BrowserWindow {
-        id: number;
+        readonly id: number;
```

Comment:
Let's delete this line and use the definition from electron.d.ts (just built locally and that's fine)

Reply:
done